src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -584,4 +584,16 @@
   <data name="PatchResourceMustBeJson" xml:space="preserve">
     <value>The resource to patch must be in JSON format.</value>
   </data>
+  <data name="ReindexingUserAction" xml:space="preserve">
+    <value>Resubmit the same reindex job to finish indexing the remaining resources.</value>
+  </data>
+  <data name="ReindexingResourceVersionConflict" xml:space="preserve">
+    <value>One or more resources failed to reindex due to a non-matching version.</value>
+  </data>
+  <data name="ReindexingServiceUnavailable" xml:space="preserve">
+    <value>Failed to reindex resource because the Cosmos service was unavailable.</value>
+  </data>
+  <data name="ReindexingSqlError" xml:space="preserve">
+    <value>Error from SQL database on reindex.</value>
+  </data>
```

Comment:
I have feeling what it's worth to put strings to resource files if they are user facing one.

These two are for emitting message to a log, and I don't think it's worth to put them in resources.

Reply:
+1 I agree. Removed the string resources in 57a3a31